docs/changelog/90174.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 90174
+summary: Add missing Disk Indicator health api IDs
+area: Health
+type: enhancement
```

Comment:
Probably shouldn't be an enhancement since it's a bugfix to a new indicator right? Maybe a non-issue?